grafana/Tiltfile

```diff
@@ -72,14 +72,16 @@ def get_or_create(dict, key):
     return dict[key]
 
 
-def grafana(context, plugin_files, grafana_version='latest', namespace='grafana', deps=[], extra_env={}):
+def grafana(context, plugin_files, grafana_image='grafana/grafana', grafana_version='latest', namespace='grafana', license='', deps=[], extra_env={}):
```

Comment:
This is potentially a breaking change. Let's make sure we bump the version correctly when we release it.